test/Sentry.Unity.iOS.Tests/IosNativeScopeObserverTests.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Globalization;
+using NUnit.Framework;
+using Sentry.Infrastructure;
+
+namespace Sentry.Unity.iOS.Tests
+{
+    public class IosNativeScopeObserverTests
+    {
+        private class SerializationTestClass
+        {
+            public string Member => throw new NullReferenceException();
+        }
+
+        private class Fixture
+        {
+            public SentryUnityOptions Options { get; set; } = new();
+            public IosNativeScopeObserver GetSut() => new(Options);
+        }
+
+        private Fixture _fixture = new();
+
+        [SetUp]
+        public void SetUp() => _fixture = new Fixture();
+
+        [Test]
+        public void GetTimestamp_ReturnStringConformsToISO8601()
+        {
+            var timestamp = SystemClock.Clock.GetUtcNow();
```

Comment:
SystemClock is used when we need to control which time is used at different places for testing.
Here you just need a timestamp, better to stick to the framework's API:

```suggestion
            var timestamp = DateTimeOffset.UtcNow;
```